source/agora/crypto/Hash.d

```diff
@@ -92,6 +92,15 @@ public alias HashDg = void delegate(in ubyte[]) @safe pure nothrow @nogc;
 private enum hasComputeHashMethod (T) = is(T == struct)
     && is(typeof(T.init.computeHash(HashDg.init)));
 
+/// Magic value hashed together with all user data
+private ulong magic;
+
+/// Sets the magic value
+public void setHashMagic (ulong val) @safe nothrow @nogc
```

Comment:
We don't use `setHashMagic` anywhere and don't have any test cases for the function. How about using `setHashMagic` now in our code or adding a unit test for that?

Reply:
Pushed a test as well, thanks!